src/views/app/detail-panel/preview/parts/message-contact-list.tsx

```diff
@@ -215,7 +221,7 @@ const MessageContactList: FC<{
 			</Container>
 			<Container ref={containerRef} width="fit" mainAlignment="flex-start">
 				{message.urgent && <Icon data-testid="UrgentIcon" color="error" icon="ArrowUpward" />}
-				{showBadge && (
+				{showBadge && messageFolder?.name && (
```

Comment:
The condition on messageFolder.name is redundant since it is already included in showBadge